libs/cart/src/selectors/cart/cart.selector.ts

```diff
@@ -62,12 +62,13 @@ export interface DaffCartStateMemoizedSelectors<
 
 const createCartSelectors = <
   T extends DaffCart = DaffCart,
-  V extends DaffCartOrderResult = DaffCartOrderResult
+	V extends DaffCartOrderResult = DaffCartOrderResult,
+	U extends DaffCartItem = DaffCartItem
```

Comment:
Add this generic to `getCartSelectors` as well.

Reply:
done